ttnn/cpp/ttnn/operations/eltwise/binary/binary_composite.hpp

```diff
@@ -52,8 +52,56 @@ struct ExecuteBinaryCompositeOpsIsClose
         const bool equal_nan,
         const std::optional<MemoryConfig>& memory_config = std::nullopt)
         {
-            auto op_type = get_function_type2<binary_comp_op_type>();
-            return op_type(input_tensor_a, input_tensor_b, rtol, atol, equal_nan, memory_config);
+            auto op_type = get_binary_isclose<binary_comp_op_type>();
+            return op_type(input_tensor_a, input_tensor_b, rtol, atol, equal_nan, memory_config.value_or(input_tensor_a.memory_config()));
+        }
+};
+
+template <BinaryCompositeOpType binary_comp_op_type>
+struct ExecuteDivLikeOps
+{
+    static Tensor execute_on_worker_thread(
+        const Tensor& input_tensor_a,
+        const Tensor& input_tensor_b,
+        const std::optional<MemoryConfig>& memory_config = std::nullopt)
+        {
+            auto op_type = get_binary_div_like_ops<binary_comp_op_type>();
+            return op_type(input_tensor_a, input_tensor_b, memory_config.value_or(input_tensor_a.memory_config()));
+        }
+
+    static Tensor execute_on_worker_thread(
+        const Tensor& input_tensor_a,
+        float value,
+        const std::optional<MemoryConfig>& memory_config = std::nullopt)
+        {
+            auto op_type = get_binary_div_like_ops_overload<binary_comp_op_type>();
+            return op_type(input_tensor_a, value, memory_config.value_or(input_tensor_a.memory_config()));
+        }
+};
+
+template <BinaryCompositeOpType binary_comp_op_type>
+struct ExecuteBinaryCompositeOpsDiv
+{
+    static Tensor execute_on_worker_thread(
+        const Tensor& input_tensor_a,
+        const Tensor& input_tensor_b,
+        bool accurate_mode = false,
+        string round_mode = "None",
```

Comment:
if 'string' compiles it means there is hidden 'using namespace std' somewhere in the headers and it must be removed.
cc @eyonland

Reply:
will be addressed